### PR TITLE
Add Styling with less in the guidelines

### PR DESCRIPTION
**Pull request by sbrunner**
Add example and UI metadata in the guideline

### CODE REVIEW

File: docs/guidelines.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ apply when you want to add a new feature.
   - [Use native javascript object methods instead](#use-native-javascript-object-methods-instead)
   - [Declaring an event](#declaring-an-event)
 - [Watch your watchers!](#watch-your-watchers)
+- [Styling with less](#styleing-with-less)
```
Comment (pgiraud) [inline docs/guidelines.md]: #styling-with-less ?

File: docs/guidelines.md [hunk 2]
```diff
@@ -836,3 +851,12 @@ provides a very good overview.
 ngeo [includes](../utils/watchwatchers.js) a JavaScript script that can be used
 to watch the number of watchers in an Angular application. Look at this file to
 know how to use it.
+
+## Styling with less
+
+To be able to do calculation directly with less we encourage to use a subset of the CSS units.
```
Comment (tsauerwein) [inline docs/guidelines.md]: "calculation**s**"

File: docs/guidelines.md [hunk 3]
```diff
@@ -836,3 +851,12 @@ provides a very good overview.
...
+We choose units that don't depends on the parent tags and are relative.
```
Comment (tsauerwein) [inline docs/guidelines.md]: "... that don't depend on parent tags ..."

File: docs/guidelines.md [hunk 4]
```diff
@@ -836,3 +851,12 @@ provides a very good overview.
...
+
+ * rem: font size of the root element.
+ * vw: viewport’s width, 1/100th of the width of the viewport.
```
Comment (tsauerwein) [inline docs/guidelines.md]: 1 vw is 1/100th of the width of the viewport?

File: docs/guidelines.md [hunk 5]
```diff
@@ -160,11 +161,25 @@ This plugin provides the `@ngdoc <type>` and `@ngname <name>` tags.
 `@ngdoc` is used to define the Angular type (directive, service, controller
 or filter) and `@ngname` defines the name used to register this component.
 
-For directive the HTML attributes are declared with `@htmlAttribute {<type>} <name> <description>.`.
+For directive the used HTML attributes are declared with `@htmlAttribute {<type>} <name> <description>.`.
```
Comment (tsauerwein) [inline docs/guidelines.md]: "For directive**s**"

File: docs/guidelines.md [hunk 6]
```diff
@@ -160,11 +161,25 @@ This plugin provides the `@ngdoc <type>` and `@ngname <name>` tags.
...
+
+The used metadata should be documented.
+
+The directive should provide an example.
```
Comment (tsauerwein) [inline docs/guidelines.md]: Maybe: "The usage of a directive should be shown with an example."

File: docs/guidelines.md (hunk 1 above)
Comment (sbrunner) [inline docs/guidelines.md]: Thanks, fixed

File: docs/guidelines.md (hunk 4 above)
Comment (sbrunner) [inline docs/guidelines.md]: Thanks @tsauerwein I follow all your suggestions.

**Comment by tsauerwein** [comment]
Feel free to merge

**Comment by sbrunner** [comment]
Thanks @tsauerwein :-)

**Pull request merged**